src/main/pg/adc.h

```diff
@@ -31,6 +31,9 @@
 typedef struct adcChannelConfig_t {
     bool enabled;
     ioTag_t ioTag;
+#if defined(STM32H7)
+    int8_t device; // ADCDevice
```

Comment:
How can this be set in CLI?

Reply:
I think on the targets I was using for testing I never needed to change it so it got overlooked.  I'll address this with some additional commits.

Reply:
The scope of changes to support per-channel ADC device configuration via surpasses the immediate requirements to allow working H7 targets.  Probably the long term solution is to expose 'device' in the adcchannel and adcoperating configs, remove the cli`adc_device` config setting and add some other way of specifying a device for the channel.

For now only the H7 support is *required*, other MCUs don't need it yet.

My feeling is to improve this as time permits and in the short term H7 target authors just need to be aware of this feature.  I am on-hand for technical assistance.

Reply:
For future discussion, I created a commit showing the scope of some of the changes required to allow a more flexible ADC device configuration - https://github.com/hydra/betaflight/commit/5b095a5f7574dc97ebcc2b00439372b087e08536

From the direction of that commit you can see it requires significant time and testing, there are also unanswered questions and technical issues regarding the DMA configuration which is also impacted by that idea.

Reply:
Getting and setting of this in CLI is still missing, breaking this for non-legacy Unified Targets. Seeing that there is no intention to add more legacy targets, there is no value in adding this unless it is working properly in for non-legacy targets.

Reply:
Currently, the targets that need it are the 5 new EXST targets:  SPRACINGH7CINE, SPRACINGH7NP, SPRACINGH7EF, SPRACINGH7RF, SPRACINGH7CL.  As these are all EXST targets this doesn't impact non-legacy targets and there is no support currently for unified targets that use EXST.  Precedent has already been set in the project with regards to EXST support vs unified target support and that EXST targets sometimes require additional code to support them.

Suggest merging this is, and then planning for EXST unified target support in a later release.

These new targets are in my other PRs.

Reply:
consensus is to deferred cli support to 4.4